db/migrate/20220113212542_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.references :author, :post
```

Comment:
- [ ] I suggest you to adjust this line of code like the following, just to have it working the way the ERD is writtten

```suggestion
      t.references :author, foreign_key: { to_table: 'users' }
      t.references :post, foreign_key: true
```

Reply:
Thanks, I have updated them.